### DISCUSSION TITLE
Shall we try designing a new logo?

### DISCUSSION THREAD

**Discussion by ranbir7** [Autostronomy/AstroPhot#34] [Ideas]
I have got some little experience in logo designing
Shall I try some new logo for this in future? After solving some issues?

**Comment by ConnorStoneAstro** [comment]
I like the current logo, but if you have some ideas on how to make it look better I'm happy to see them. Still I would want it to look similar to the current one.

**Reply by ranbir7** [reply]
Okay I'll keep it in mind 😄
